en/releases/1.12.md

```diff
@@ -5,26 +5,65 @@
 
 ## Changes
 
+* [Common](#common)
+* [MAVLink](#mavlink)
+* [Commander](#commander)
 * [Multicoper](#multicopter)
+* [VTOL](#vtol)
+* [Control & Navigation](#control)
+
+### Common
+
+* **Remaining Flight Range based RTL Trigger ([PR#16399](https://github.com/PX4/PX4-Autopilot/pull/16399))**
+  * Calculates time to home, on RTL, taking into account vehicle speed, wind speed, and desination distance/direction
+* Pre-emptive geofence breach ([PR#16400](https://github.com/PX4/PX4-Autopilot/pull/16400))
+  * Triggers a breach if the _predicted_ current trajectory will result in a breach, allowing the vehicle to be re-routed to a safe hold position.
+
+### MAVLink
+
+* **MAVLink Ethernet Configuration ([PR#14460](https://github.com/PX4/PX4-Autopilot/pull/14460))**
+  * MAVLink Ethernet channel settings such as UDP port, remote port and broadcast mode now can be changed dynamically via parameters.
+* **Support for COMPONENT_INFORMATION ([PR#16039](https://github.com/PX4/PX4-Autopilot/pull/16039))**
+  * This new message enables a GCS to understand which commands are supported by any given Autopilot (RFC: [mavlink#1339](https://github.com/mavlink/mavlink/issues/1339))
+
+### Commander
+
+* **Commander: use control mode flags and cleanup arm_disarm ([PR#16266](https://github.com/PX4/PX4-Autopilot/pull/16266))**
+  * Consolidates arming logic, and does a better job at keeping the current state
+* **Commander: Separete out manual control setpoint processing ([PR#16878](https://github.com/PX4/PX4-Autopilot/pull/16878))**
+  * Adds a new class `ManualControl` for handling `manual_control_setpoint` and handles RC loss, RC override, and RC arming/disarming
 
 ### Multicopter
 
-* #### Intuitive stick feel in Position mode
+* **More intuitive stick feel in Position mode**
   * Horizontal stick input mapped to acceleration instead of velocity setpoints
   * Removes unexpected tilt changes upon reaching travel speed velocity
   * Intuitive shunting e.g. when landing
-  * Comparable to Altitude mode but with the familiar benefits
```

Comment:
This is a description of existing position mode when compared to altitude mode, so what is the point?

Reply:
Most of this is not relevant to document other than change from speed to acceleration on sticks. That is done.